spyder/plugins/completion/kite/client.py

```diff
@@ -114,7 +114,9 @@ def send(self, method, params, url_params):
                 try:
                     success, response = self.perform_http_request(
                         http_verb, path, params)
-                except (ConnectionRefusedError, ConnectionError):
+                # TODO: Define these exceptions
+                # except (ConnectionRefusedError, ConnectionError):
```

Comment:
These errors exist in Python 3 but not in Python 2. In Python 2 both are subclasses of OSError, so please add them to our `py3compat` module (at the end of that file you would find a list of compatibility errors).

Reply:
I mean, you can add to py3compat like this:

```python
if PY2:
    ConnectionRefusedError = ConnectionError = OSError
else:
    ConnectionRefusedError = ConnectionRefusedError
    ConnectionError = ConnectionError
```

Reply:
Right, will fix now!